uobtheatre/payments/payables.py

```diff
@@ -151,59 +160,133 @@ def refund(self, authorizing_user: User, do_async=True, send_admin_email=True):
                 html_message=mail.to_html(),
             )
 
-    def complete(self, payment: Transaction = None):
+    @property
+    def total(self) -> int:
+        """The total cost of the payable in pence.
+
+        The final price of the payable with all dicounts and misc costs
+        applied. This is the price the User will be charged.
+
+        Returns:
+            (int): total price of the payable in penies
         """
-        Called once the pay object has been completly paid for. Payment passed is the finishing transaction
+        subtotal = self.subtotal
+        if subtotal == 0:  # pylint: disable=comparison-with-callable
+            return 0
+        return math.ceil(subtotal + self.misc_costs_value)
```

Comment:
Kinda feel like this should be one-lined
```suggestion
        return 0 if self.subtotal == 0 else math.ceil(self.subtotal + self.misc_costs_value)
```